src/redux/reducers/index.js

```diff
@@ -109,25 +106,28 @@ export const dateRangeFilterReducer = (state = preloadSelectedTimelineRange, act
   }
 };
 
-const preloadCollections = {};
+// this same uuid recycled across logins -- which is only development?
+const defaultCollectionId = uuidv4();
+
+const createDefaultCollections = () => {
+  const timeCreated = new Date();
+  return {
+    [defaultCollectionId]: {
+      created: timeCreated,
+      lastUpdated: timeCreated,
+      label: 'Untitled Collection',
+      resourceIds: {},
+      lastAddedResourceId: null,
+    },
+  };
+};
+
+const preloadCollections = createDefaultCollections();
```

Comment:
Feels strange that we are creating the default collection before the user logs in. Ideally a default collection is created when the patient logs in and no collections exist yet. 

In practice, I guess this way works too since we create a new default collection on logout. But this probably will need to get updated again when we persisting Collections data.

Reply:
if persistence were implemented, I'm not sure whether the persisted, "default collection" would be created ^here.

...it might exist in dedicated middleware.